internal/linker/linker.go

```diff
@@ -23,7 +23,8 @@ import (
 )
 
 const (
-	MagicValueEnv = "GARBLE_LINKER_MAGIC"
+	MagicValueEnv     = "GARBLE_LINKER_MAGIC"
+	RemoveFuncNameEnv = "GARBLE_LINK_REMOVE_FUNC_NAMES"
```

Comment:
Wouldn't it be easier to just have a general `GARBLE_TINY` env var, because we will probably add more linker stage obfuscations.